src/hooks/chart.ts

```diff
@@ -269,11 +279,26 @@ export const useChartData = (plot: Plot) => {
     isFetchSelected,
     setIsFetchSelected,
   } = useChartSelectGrid({
```

Comment:
Can you change this hook name as well? We want to rename it to `useChartControlsGrid`